src/screens/SurveyDetailScreen.tsx

```diff
@@ -80,64 +84,75 @@ export const SurveyDetailScreen = ({ route }: Props) => {
 
   return (
     <SafeAreaViewFlex>
-      <ScrollView refreshControl={RefreshControl}>
-        <WrapperWithOrientation>
-          {!!shownTitle?.length && <SectionHeader title={shownTitle} />}
-          <DateComponent start date={survey.date.dateStart} />
-          <DateComponent date={survey.date.dateEnd} />
-          {!!survey.description?.[languages[0]]?.length && (
-            <Wrapper style={styles.noPaddingBottom}>
-              <RegularText>{survey.description[languages[0]]}</RegularText>
-              {!!survey.description?.[languages[1]]?.length && (
-                <RegularText italic>{survey.description[languages[1]]}</RegularText>
-              )}
-            </Wrapper>
-          )}
-          {!!survey.questionTitle?.[languages[0]]?.length && !!title?.length && (
-            <Wrapper style={styles.noPaddingBottom}>
-              <BoldText>{survey.questionTitle?.[languages[0]]}</BoldText>
-              <BoldText italic>{survey.questionTitle?.[languages[1]]}</BoldText>
-            </Wrapper>
-          )}
-          {survey.responseOptions.map((responseOption, index) => (
-            <SurveyAnswer
-              archived={archived}
-              faded={!!selection && selection !== responseOption.id}
-              index={index}
-              responseOption={responseOption}
-              selected={selection === responseOption.id}
-              setSelection={setSelection}
-              key={responseOption.id}
-            />
-          ))}
-          {!archived && (
-            <>
-              <Wrapper>
-                <Button
-                  disabled={
-                    !selection || (!!previousSubmission && selection === previousSubmission)
-                  }
-                  title={buttonText}
-                  onPress={submitSelection}
-                />
+      <DefaultKeyboardAvoidingView>
+        <ScrollView refreshControl={RefreshControl} ref={scrollViewRef}>
```

Comment:
the scrollview should have added `keyboardShouldPersistTaps="handled"` inside the `DefaultKeyboardAvoidingView` (like in FormScreen for example) to be able to click the button with open keyboard

Reply:
with adding a `Keyboard.dismiss();` to the button

Reply:
done